src/actions/projectActions.js

```diff
@@ -40,29 +29,20 @@ export function syncUserProj(payload) {
 export const asyncExampleProj = () => {
     // fetch example projects
     return (dispatch) => {
-        let exampleVals = [];
-        scenes.where("uid", "==", "1").get().then(snap => {
-            let num_proj_loaded = 0;
-            snap.forEach(doc => {
-                storageRef.child(`/images/perspective/${doc.id}`)
-                    .getDownloadURL()
-                    .catch(() => {
-                        console.error("Error: Missing preview image");
-                    })
-                    .then((img) => {
-                        let dat = doc.data();
-                        exampleVals.push({
-                            name: dat.name,
-                            id: doc.id,
-                            data: dat,
-                            url: img ? img : "/img/no_preview.jpg"
+        fetch(`${sceneRef}/`, {headers: {"x-access-token": "1"}}).then((response) =>{
+            if(response.status === 200){
+                response.json().then((json) =>{
+                    for(let i = 0; i < json.length; ++i){
+                        let id = json[i].firebaseID ? json[i].firebaseID : json._id;
```

Comment:
The ideal thing to do here would be to have the migration script change all of the image names to the new mongo id. This way we don't have any conflicts later.

Reply:
Will update, seems I also forgot to handle the example scene images, since they're still being grabbed from firebase.